content/blog/2019/01-09-state-of-inexor-2018.md

```diff
@@ -0,0 +1,44 @@
+---
+layout:     post
+title:      State of Inexor 2018
+date:       2019-01-09 17:00:00 +0200
+author:     Fohlen
+summary:    Current state of Inexor in 2018 and outlook to 2019.
+---
+
+Howdy folks! It has been mere two months since the last official news from our project, so I'll briefly cover what happened:
+
+- we've had another amazing hackathon in Stuttgart
+- decision has been made to re-start the project
```

Comment:
restart is one word.

Reply:
"restart" is maybe also a bit too harsh. The ideas and concepts will live on. It might also be important to note that we did not decide light-heartedly